lib/Gedmo/SoftDeleteable/SoftDeleteableListener.php

```diff
@@ -29,6 +29,13 @@ class SoftDeleteableListener extends MappedEventSubscriber
      */
     const POST_SOFT_DELETE = "postSoftDelete";
 
+    /**
+     * Objects soft-deleted on flush.
+     *
+     * @var array
+     */
+    private $softDeletedObjects = [];
```

Comment:
please use basic array here, since extension compatibility is with php 5.3, everything else seems OK

Reply:
`= array();` I mean ;)